Cargo.toml

```diff
@@ -101,6 +101,7 @@ smallvec = { version = "1.13.2", features = [
     "const_new",
     "union",
 ] }
+streampager = "0.10.3"
```

Comment:
Perhaps, "CC0-1.0" can be allowed, but I haven't updated the list yet.
https://github.com/martinvonz/jj/actions/runs/10224603116/job/28292547146?pr=4203#step:4:83